build/config/BUILDCONFIG.gn

```diff
@@ -543,7 +543,11 @@ if (custom_toolchain != "") {
   import("//build/config/ios/ios_sdk.gni")  # For use_ios_simulator
   host_toolchain = "//build/toolchain/mac:clang_$host_cpu"
   if (use_ios_simulator) {
-    set_default_toolchain("//build/toolchain/mac:ios_clang_x64")
+    if (target_cpu == "arm" || target_cpu == "arm64") {
```

Comment:
Does this work for armv7 builds? IIRC they use soft-fp ABI.

Reply:
simulator doesn't actually use armv7, as the only possible hosts are ARM64 (on M1) or x86_64. I can remove the arm condition.

Reply:
sgtm -- that would make it clearer for readers.